flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/utils/ParquetSchemaConverter.java

```diff
@@ -101,11 +110,32 @@ private static Type convertToParquetType(
             case TIMESTAMP_WITH_LOCAL_TIME_ZONE:
                 return Types.primitive(PrimitiveType.PrimitiveTypeName.INT96, repetition)
                         .named(name);
+            case ARRAY:
+                ArrayType arrayType = (ArrayType) type;
+                return ConversionPatterns.listOfElements(
+                        repetition,
+                        name,
+                        convertToParquetType(LIST_ELEMENT_NAME, arrayType.getElementType()));
+            case MAP:
+                MapType mapType = (MapType) type;
+                return ConversionPatterns.stringKeyMapType(
```

Comment:
must be string key?

Reply:
supports multiple types in latest commit.